advanced_functionality/multi_model_xgboost_home_value/xgboost_multi_model_endpoint_home_value.ipynb

```diff
@@ -338,6 +338,7 @@
     "from sagemaker import get_execution_role\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #27.    ENDPOINT_NAME = f'mme-xgboost-housing-{strftime("%Y-%m-%d-%H-%M-%S", gmtime())}'
</pre><p>Please run <code>black-nb -l 100 {notebook-name}.ipynb</code> for each notebook to apply code formatting - the standard is double quotes. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3439/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Thanks, I used this command and with new commits it passed the code-formatting check</p>